site/docs/Learn-Schema.md

```diff
@@ -226,17 +231,17 @@ For example, here are some types that might implement `Character`:
 ```graphql
 type Human implements Character {
   id: String!
-  name: String
+  name: String!
```

Comment:
I think it's probably better to avoid overuse of !. For one, it's noisy when teaching graphql concepts, but also it's a dangerous default state when built on languages that can throw errors